docs/getting_started.md

```diff
@@ -8,3 +8,12 @@
 Create an account with sufficient permissions to be able to merge on protected branches using the workflow features.
 
 **Supported host:** GitHub
+
+## Configuration
+
+### Build
+
+You should create a hook that is called after merge on the first step of your workflow (e.g. master).
+This will create the docker image by calling the [build API](./api.md#api-build-api).
+The created image docker will be tagged with a reference to the version of your application and the commit hash.
+**This will guarantee that you are using the same version for all environment.**
```

Comment:
What about ?
```suggestion
You should create a hook triggered every time new commits are merged on the first step of your workflow (e.g. master) calling [Nestor's build API](./api.md#api-build-api).
This will create a new docker image corresponding to this new version. Nestor will tag this image with a reference to the version of your application and the commit hash before adding it to your Docker registry.
**This will guarantee that you are using the same version for all environment.**
```
You could also try to add some spacing because everything is rendered on a single line.